examples/guides/se-restful-webservice/README.adoc

```diff
@@ -642,31 +547,31 @@ These link the health-related endpoints your code is exposing to the new methods
 
 // tag::tryReadiness[]
 
-=== Try the readiness check
-Access the readiness check endpoint:
+=== Check the server's health
+Run this command
 [source,bash]
-curl -i -X GET http://localhost:8080/ready
+curl -i -X GET http://localhost:8080/health
 
-Our readiness check returns no payload, just the 200 status, so you won't see any data
-displayed. The `-i` option shows the
-200 status in the response.
-// end::tryReadiness[]
-
-// tag::tryLiveness[]
+and you should see output like this
+[listing,subs=+quotes]
+----
+{"outcome":"UP","checks":[{"name":"deadlock","state":"UP"},{"name":"diskSpace","state":"UP","data":{"free":"180.64 GB","freeBytes":193955860480,"percentFree":"38.79%","total":"465.72 GB","totalBytes":500068036608}},*{"name":"greetingAlive","state":"UP"}*,{"name":"heapMemory","state":"UP","data":{"free":"230.86 MB","freeBytes":242074232,"max":"4.00 GB","maxBytes":4294967296,"percentFree":"99.39%","total":"256.00 MB","totalBytes":268435456}}]}
+----
```

Comment:
Would it be possible to format the JSON in the rendered doc, so that readers can parse the fields easily?

Reply:
Possibly (we might need to do the formatting manually in the `README.adoc` file) but that is not how the output is displayed when users run the command.

Reply:
It turns out that we can use `curl GET http://localhost:8080/health | python -m json.tool` to format the output nicely. I've made that change in the SE file, and the MP file includes that part anyway.